### PR TITLE
Add AICOE Config

### PR DESCRIPTION
**Pull request by saisankargochhayat**
## Related Issues and Dependencies
Add CI to enable testing on test env
Closes : #603

### CODE REVIEW

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+check:
+  - thoth-precommit
+  - thoth-build
+build:
+  base-image: "registry.access.redhat.com/ubi8"
```
Comment (saisankargochhayat) [inline .aicoe-ci.yaml:5]: Is the base image necessary in docker file way to build?

File: .aicoe-ci.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  build-stratergy: Dockerfile # Allowed values: Source, Dockerfile, Containerfile (default: Source)
+  dockerfile-path: Dockerfile
+  custom_tag: latest # custom tag to be push to registry
```
Comment (saisankargochhayat) [inline .aicoe-ci.yaml:8]: Also, do I let this be 'latest'?

**Comment by saisankargochhayat** [comment]
@harshad16 could you please have a look?

File: .aicoe-ci.yaml (hunk 1 above)
Comment (harshad16) [reply .aicoe-ci.yaml:5]: No, it is not.please remove it.

File: .aicoe-ci.yaml (hunk 2 above)
Comment (harshad16) [reply .aicoe-ci.yaml:8]: we normally keep only tag version, but we can have the kebechet latest tag as well.

Comment (saisankargochhayat) [reply .aicoe-ci.yaml:8]: So just remove custom_tag key?

Comment (saisankargochhayat) [reply .aicoe-ci.yaml]: Thanks I have removed it

File: .aicoe-ci.yaml (hunk 1 above)
Comment (saisankargochhayat) [reply .aicoe-ci.yaml]: Thanks

**Review by harshad16 — APPROVED** [review]
lgtm , thanks :100:

**Pull request merged**